R/deprecated.R

```diff
@@ -131,7 +520,7 @@ lognorm_dist_def <- function(mean, mean_sd,
                              to_log = FALSE) {
   lifecycle::deprecate_warn(
     "1.5.0", "lognorm_dist_def()", "LogNormal()",
-    "The function will be removed completely in version 2.0.0."
+    "The function will be removed completely in the next version."
```

Comment:
Are we sure we won't have other releases before v2.0.0?

Reply:
I think we will which is why I decided to change these - so we can don't have to wait for a certain future version in order to be able to hard deprecate.